src/test/java/org/gridsuite/explore/server/ExploreTest.java

```diff
@@ -77,6 +79,7 @@ public class ExploreTest {
     public static final String FILTER = "FILTER";
     private Map<String, Object> specificMetadata = new HashMap<>();
     private Map<String, Object> specificMetadata2 = new HashMap<>();
+    private Map<String, Object> caseSpecificMetadata = new LinkedHashMap<>();
```

Comment:
why it has to be a `LinkedHashMap` instead of a `HashMap` like other metadata members here ?